src/app/games/mancala/ba-awa/BaAwaTutorial.ts

```diff
@@ -69,18 +69,18 @@ export class BaAwaTutorial extends Tutorial {
                 [2, 7, 2, 0, 0, 0],
             ], 0, [0, 0]),
             [MancalaMove.of(MancalaDistribution.of(0))],
-            $localize`There it is, the opponent captured 4 seeds.`,
+            $localize`There it is, you captured 4 seeds.`,
             $localize`Wrong, you did not capture anything.`,
         ),
         TutorialStep.fromMove(
             $localize`End game`,
-            $localize`At any end of turn, if the number of seed reach 8 or less, the first player capture capture the 8 seeds`,
+            $localize`At any end of turn, if the number of seed reach 8 or less, the first player capture capture the 8 seeds.<br/><br/>You're playing Dark, end the game by capturing!`,
```

Comment:
cet exemple contient DÉJÀ seulement 8 pièces, ce qui veut dire que le jeu aurait du s'arrêter avant ce tour
Peut-être préciser, "the first player (Dark) ..."